dependencies.yaml

```diff
@@ -287,7 +287,7 @@ dependencies:
   org.postgresql:
     postgresql:
       lang: java
-      version: "9.4.1212"
+      version: "42.2.6"
```

Comment:
Whoa. That's quite a jump. :open_mouth:

Reply:
they went from 9.x to 42.x 🤷‍♂

Reply:
This makes no sense.

_/me goes back working on features for 100.13.21_